meetings/2020-02-25.md

```diff
@@ -68,9 +68,8 @@
 * Add Foundation-wide copyright guidance [#414](https://github.com/openjs-foundation/cross-project-council/pull/414)
   * No update, still in progress
 
-
 * Add post-graduation checklist [#386](https://github.com/openjs-foundation/cross-project-council/pull/386)
-  *
+  * No update, still in progress
```

Comment:
Is this intended to be here?

Reply:
I added it accidentally in managing a couple PRs, but it is relevant/accurate, so I left it there.